src/analysis_app.py

```diff
@@ -293,7 +293,8 @@ def analysis_to_ent_result(self, doc_analysis):
                 ent_res = {
                     'type': ent['type'],
                     'text': ent['text'],
-                    'score': ent['score']
+                    'score': ent['score'],
+                    'entityUid': ent.get('uid')
```

Comment:
shouldn't it be `ent.get('uid','')`? What happens when you get `None` there?

Reply:
We are using the `csv.DictWriter` which handles `None` values correctly, so this is not a problem.